website/docs/r/palo_alto_local_rulestack_rule.html.markdown

```diff
@@ -28,6 +28,7 @@ resource "azurerm_palo_alto_local_rulestack_rule" "example" {
   name         = "example-rule"
   rulestack_id = azurerm_palo_alto_local_rulestack.example.id
   priority     = 1000
+  action = "Allow"
```

Comment:
I believe the linter will complain that these aren't aligned:

```suggestion
  priority     = 1000
  action       = "Allow"
```

You should be able to run `make website-lint` to fix that fwiw